.circleci/config.yml

```diff
@@ -207,6 +207,10 @@ commands:
           command: |
             ROOT=$PWD
             cd build/docker
+            if [[ "<<parameters.platform>>" == "centos7" ]]; then
+                sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+                sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
+            else
```

Comment:
Shouldn't we do this in "readies" instead? https://github.com/RedisLabsModules/readies/blob/master/bin/getclang#L114